caesars-palace/blocks/slider-card/slider-card.css

```diff
@@ -120,12 +133,62 @@
     position: relative;
     padding: 16px;
     display: block;
-    height: 190px;
   }
 
   .slider-card .card .close-button {
-    margin: -27px 20px 20px;
+    margin: 0 20px 20px;
     position: relative;
     z-index: 2;
   }
+
+
+  .slider-card-wrapper .chevron-left {
```

Comment:
Instead of using the name `chevron` I think it would be better to have one class called `slider-button` that both the loeft and right buttons could share. Then you could add `left` and `right` modifiders to perform the unique styling for both.